Mathlib/NumberTheory/ArithmeticFunction.lean

```diff
@@ -423,41 +456,44 @@ theorem zeta_apply_ne {x : ℕ} (h : x ≠ 0) : ζ x = 1 :=
   if_neg h
 #align nat.arithmetic_function.zeta_apply_ne Nat.ArithmeticFunction.zeta_apply_ne
 
-@[simp]
+-- Porting note: removed `@[simp]` to make the linter happy.
```

Comment:
What did the linter say?

Reply:
/- The `simpNF` linter reports:
SOME SIMP LEMMAS ARE NOT IN SIMP-NORMAL FORM.
see note [simp-normal form] for tips how to debug this.
https://leanprover-community.github.io/mathlib_docs/notes.html#simp-normal%20form -/
#check @Nat.ArithmeticFunction.coe_zeta_smul_apply /- Left-hand side simplifies from
  ↑(↑Nat.ArithmeticFunction.zeta • f) x
to
  ∑ x in Nat.divisorsAntidiagonal x, (if x.fst = 0 then 0 else 1) • ↑f x.snd
using
  simp only [@Nat.ArithmeticFunction.smul_apply,
 @Nat.ArithmeticFunction.natCoe_apply,
 @Nat.ArithmeticFunction.zeta_apply,
 @Nat.cast_ite,
 @Nat.cast_zero,
 @Nat.cast_one]
Try to change the left-hand side to the simplified term!
 -/

 Should I follow the suggestion of the linter?